PlaceRouteHierFlow/PnRDB/ReadConstraint.cpp

```diff
@@ -259,6 +261,8 @@ void PnRdatabase::ReadConstraint_Json(PnRDB::hierNode& node, const string& jsonS
               temp_pair.second = k;
             }
           }
+          if (temp_pair.first == -1) logger->error("Block {0} not found", pair["block1"]);
```

Comment:
If a block specified in constraint not found in netlist, an error will be output.